### ISSUE TITLE
Amazon settings error

### ISSUE THREAD

**Issue by pacahon** [django-dbbackup/django-dbbackup#125]
Hi, I guess there is an error (lines 98-99)
https://github.com/django-dbbackup/django-dbbackup/blob/master/dbbackup/settings.py#L99
Traceback:
```
AttributeError: 'Settings' object has no attribute 'BUCKET'
```

**Comment by pacahon** [comment]
And it may be necessary to update the documentation on Read The Docs! Thanks.

**Comment by ZuluPro** [comment]
This issue will be closed by PR #126

**Comment by ZuluPro** [comment]
@pacahon : Do you still have an error with this patch ?
